src/macros/GroundOverlay.jsx

```diff
@@ -0,0 +1,86 @@
+/* global google */
+import React from "react"
+import PropTypes from "prop-types"
+
+import {
+  construct,
+  componentDidMount,
+  componentDidUpdate,
+  componentWillUnmount,
+} from "../utils/MapChildHelper"
+
+import { MAP, GROUND_LAYER } from "../constants"
+
+export const __jscodeshiftPlaceholder__ = `{
+  "eventMapOverrides": {
+    "onDblClick": "dblclick"
+  },
+  "getInstanceFromComponent": "this.state[GROUND_LAYER]"
+}`
+
+/**
+ * @url https://developers.google.com/maps/documentation/javascript/reference#GroundOverlay
+ */
+export class GroundOverlay extends React.PureComponent {
+  static propTypes = {
+    __jscodeshiftPlaceholder__: null,
+    /**
+     * @type string
+     */
+    url: PropTypes.string.isRequired,
+
+    /**
+     * @see https://developers.google.com/maps/documentation/javascript/reference#GroundOverlay
+     */
+    bounds: PropTypes.object.isRequired,
```

Comment:
Don't manually include these props, they're meant to be generated automatically. Just `jscodeshiftPlaceholder`